charts/opensearch/templates/securityconfig.yaml

```diff
@@ -5,9 +5,9 @@ metadata:
   name: {{ .Values.securityConfig.config.securityConfigSecret }}
   namespace: {{ .Release.Namespace }}
   labels:
-    app: {{ .Chart.Name }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
+    app: "{{ .Chart.Name }}"
+    release: {{ .Release.Name | quote }}
+    heritage: {{ .Release.Service | quote }}
```

Comment:
Change these to the labels template. Technically `app` has been replaced by a newer label in Helm.

Reply:
Yes you're right! I didn't check twice!